tests/test_prepare_dut.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import logging
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.pretest,
+    pytest.mark.topology('util') #special marker
+]
+
+def test_prepare_dut(duthost, request):
```

Comment:
I think test_clean_dut will be a more specific name.

Reply:
Good idea. Changed both file name and function name.